### PR TITLE
docs: Add 2023-04-06 meeting notes

### PR DESCRIPTION
**Pull request by sam3k**
Closes https://github.com/eslint/tsc-meetings/issues/420

### CODE REVIEW

File: notes/2023/2023-04-06.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+**TSC Summary:** Do we want to add a requirement on bug reports for a minimal reproduction case with either:
+
+1. Playground link
+2. Stackblitz link
+3. GitHub repo?
+
+Some other projects are requiring this and it seems like it could further reduce time to evaluate and close these issues. For issues related to rules, playground links are really easy to create.
+
+**Resolution:** we've agreed to add a field to require a repro link in bug reports. I'll take the action item to update the template.
```
Comment (nzakas) [inline notes/2023/2023-04-06.md:47]: Just need to call out the action item.

```suggestion
**Resolution:** we've agreed to add a field to require a repro link in bug reports.

**Action Item:** @nzakas to update the issue template
```

**Review by nzakas — APPROVED** [review]
LGTM. Just waiting for @mdjermanovic to approve.

File: notes/2023/2023-04-06.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+**Action Item:** @mdjermanovic will submit a pull request to modify this behavior
+
+### Encourage transition to FlatConfig
+
+> @nzakas:
+>
+> Now that flat config is feature complete, do we want to announce that and encourage people to start transitioning their configs?
+
+**Resolution:** We've decided to announce flat config as feature complete. The action item is for me to write a blog post announcing it.
```
Comment (mdjermanovic) [inline notes/2023/2023-04-06.md:25]:
```suggestion
**Resolution:** We've decided to announce flat config as feature complete.

**Action Item:** @nzakas will write a blog post announcing it.
```

File: notes/2023/2023-04-06.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+[`2023-04-06-transcript.md`](2023-04-06-transcript.md)
+## Attending
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+### [Commits should create a release pull request](https://github.com/eslint/create-config/pull/56)
```
Comment (mdjermanovic) [inline notes/2023/2023-04-06.md:14]:
```suggestion
### Commits should create a release pull request
```

This topic wasn't related to the linked PR.

**Comment by mdjermanovic** [comment]
Something's wrong with the EasyCLA check. I'll close-reopen this PR to check if the problem has been fixed.

**Comment by nzakas** [comment]
I think we can just go ahead and merge this as-is. We know it's an EasyCLA error and it would be nice to finally generate the next TSC meeting issue. :)

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**